.github/CODEOWNERS

```diff
@@ -1,33 +1,17 @@
 # CODEOWNERS
 #
-# documentation for this file can be found at:
+# Documentation for this file can be found at:
 # https://help.github.com/en/articles/about-code-owners
 
 # These are the default owners for the code and will
 # be requested for review when someone opens a pull request.
-# order is alphabetical for easier maintenance.
+# Order is alphabetical for easier maintenance.
 #
 # Adarsh Kumar Yadav (adarsh-dell)
+# Abhilash Muralidhara (abhi16394)
```

Comment:
@donatwork 
Adarsh <-> Abhi
To maintain the alphabetical order?

Reply:
Fixed.